<?xml version="1.0"?>
<launch>

    <node pkg="interaccion" name="dialogo_nodo" type="dialogo_nodo" output="screen" launch-prefix="xterm -e"/>

    <node pkg="interaccion" name="empaquetador_nodo" type="empaquetador_nodo.py" output="screen" launch-prefix="xterm -e"/>

    <node pkg="interaccion" name="matematico_nodo" type="matematico_nodo.py" output="screen" launch-prefix="xterm -e"/>

    <node pkg="interaccion" name="reloj_nodo" type="reloj_nodo" output="screen" launch-prefix="xterm -e"/>
    
    <node pkg="interaccion" name="tts" type="tts.py" output="screen" launch-prefix="xterm -e"/>
    

</launch>